src/Driver/driver_configs.jl

```diff
@@ -407,12 +414,7 @@ function OceanSplitExplicitConfiguration(
 
     Q_2D = init_ode_state(dg_2D, FT(0); init_on_cpu = true)
 
-    # XXX: Needs updating for multiple polynomial orders
-    N = polynomialorders(grid_3D)
-    # Currently only support single polynomial order
-    @assert all(N[1] .== N)
-    N = N[1]
-    vert_filter = CutoffFilter(grid_3D, N - 1)
+    vert_filter = CutoffFilter(grid_3D, polyorder_vert - 1)
```

Comment:
@blallen I requested your review because I wasn't extremely sure how to handle this case for the `OceanSplitExplicitConfiguration`, since it appeared to be slightly different than the other driver configurations for the `Atmos` simulations. If we understood correctly (i.e., @thomasgibson and I) this `CutoffFilter` is applied only in the vertical, so I used the `polyorder_vert` degree here. Thanks!

Reply:
I believe this is correct, but will let @blallen weigh in.

Reply:
looks good to me!